### PR TITLE
Update active committers and support white papers part

### PR DESCRIPTION
**Pull request by christophechevalier**

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -27,11 +27,13 @@ menu: "community"
 
 *Active committers on Petals ESB and related projects.*
 
-* Alexandre Lagane (Linagora) - [@alagane](https://github.com/alagane)
-* Bertrand Escudié (Linagora) - [@bescudie](https://github.com/bescudie)
-* Christophe Chevalier (Linagora) - [@christophechevalier](https://github.com/christophechevalier)
-* [Christophe Deneux](https://www.linkedin.com/in/cdeneux/) (Linagora) - [@chrisdeneux](https://twitter.com/ChrisDENEUX) - [@cdeneux](https://github.com/cdeneux)
-* [Pierre Souquet](https://fr.linkedin.com/in/pierre-souquet-18955528) (Linagora) - [@psouquet](https://github.com/psouquet)
+* [Alexandre Lagane](https://www.linkedin.com/in/alexandrelagane) (Linagora) - [![GitHub](./resources/images/github_mark_32px.png "GitHub")](https://github.com/alagane){:target="_blank"}
```
Comment (cdeneux) [inline community.md]: Perhaps can you add the role of everyone inside the Petals team ?

File: support.md [hunk 2]
```diff
@@ -11,24 +11,31 @@ The official Petals documentation is available at [doc.petalslink.com](https://d
...
-* In French: [Petals ESB, un bus de services](resources/pdf/Livre_blanc_ESB_Petals.pdf)
-* [How to choose your process orchestration technology?](resources/pdf/white_paper_orchestration.pdf)
+* [A series of articles](https://blog.linagora.com/vision-des-esbs-par-un-architecte-dentreprise-ou-ce-que-devraient-etre-les-esbs){:target="_blank"} written by an Enterprise Architect (in French).
+* In French: [Petals ESB, un bus de services](resources/pdf/Livre_blanc_ESB_Petals.pdf){:target="_blank"}
+* [How to choose your process orchestration technology?](resources/pdf/white_paper_orchestration.pdf){:target="_blank"}
 
-Various presentations are also available on [Slide Share](https://slideshare.net/petalslink).
+Various presentations are also available on [Slide Share](https://slideshare.net/petalslink){:target="_blank"}.
+
+And somes videos conferences are also available :
```
Comment (cdeneux) [inline support.md]: "And some video conferences are also available"

**Review by cdeneux — CHANGES_REQUESTED** [review]

File: community.md (hunk 1 above)
Comment (vincent-zurczak) [reply community.md]: Why do you add `target: _blank`?
And roles can evolve. This is useless.

File: support.md (hunk 2 above)
Comment (vincent-zurczak) [reply support.md]: "And some video conferences are available".
Avoid and + also together. :)

**Review by vincent-zurczak — CHANGES_REQUESTED** [review]
The use of `target: _blank` is useless.
If people want to open links in another tab or window, they will be able to do it. No need to force the opening somewhere else.

File: community.md (hunk 1 above)
Comment (christophechevalier) [reply community.md]: This will open it in a new tab. I find it more practical when it comes to opening an external link.  
I didn't push roles ?!

**Review by vincent-zurczak — APPROVED** [review]

**Comment by vincent-zurczak** [comment]
OK for me. :+1: 
@cdeneux Waiting for your approval. If you merge, please select **squash and merge** in the drop-down list.

**Review by cdeneux — APPROVED** [review]

**Pull request merged**